locales/es-ES.json

```diff
@@ -139,18 +139,18 @@
 	"Korean": "Coreano",
 	"Norwegian": "Noruego",
 	"Polish": "Polaco",
-	"Portuguese": "Portugues",
+	"Portuguese": "Portugués",
 	"Spanish": "Español",
-	"You have to save!": "Tienes que guardar!",
+	"You have to save!": "¡Tienes que guardar!",
 	"Russian": "Ruso",
-	"Command(⌘)": "Command(⌘)",
+	"Command(⌘)": "Comando(⌘)",
 	"Editor Rulers": "Reglas del editor",
 	"Enable": "Activar",
 	"Disable": "Desactivar",
 	"Sanitization": "Saneamiento",
 	"Only allow secure html tags (recommended)": "Solo permitir etiquetas html seguras (recomendado)",
 	"Allow styles": "Permitir estilos",
 	"Allow dangerous html tags": "Permitir etiquetas html peligrosas",
-	"⚠ You have pasted a link referring an attachment that could not be found in the storage location of this note. Pasting links referring attachments is only supported if the source and destination location is the same storage. Please Drag&Drop the attachment instead! ⚠": "⚠ You have pasted a link referring an attachment that could not be found in the storage location of this note. Pasting links referring attachments is only supported if the source and destination location is the same storage. Please Drag&Drop the attachment instead! ⚠",
-	"Convert textual arrows to beautiful signs. ⚠ This will interfere with using HTML comments in your Markdown.": "Convert textual arrows to beautiful signs. ⚠ This will interfere with using HTML comments in your Markdown."
+	"⚠ You have pasted a link referring an attachment that could not be found in the location of this note. Pasting links referring attachments is only supported if the source and destination location is the same storage. Please Drag&Drop the attachment instead! ⚠": "⚠ Ha pegado un enlace a un archivo adjunto que no se puede encontrar en el almacenamiento de esta nota. Pegar enlaces a archivos adjuntos solo está soportado si el origen y el destino son el mismo almacenamiento. ¡Por favor arrastre el archivo en su lugar! ⚠",
```

Comment:
This sentence is clear in english but no so clear in spanish. A better approach would be: "Ha pegado un enlace a un archivo adjunto que no se pudo encontrar en la ubicación de almacenamiento de esta nota. Pegar enlaces a archivos adjuntos solo es posible si la ubicación del origen y del destino es la misma."

Please Drag&Drop the attachment instead! `-->` ¡Por favor mejor arrastre el archivo!
In spanish, is quite common to use the word "mejor" to imply "you should do this instead of the other". Example: "Don't eat donuts, eat fruits instead", in spanish is "No coma donas, mejor coma frutas".

Reply:
Done